PLAYER/RestorePlayerStamina.md

```diff
@@ -7,7 +7,22 @@ ns: PLAYER
 // 0xA352C1B864CAFD33 0x62A93608
 void RESTORE_PLAYER_STAMINA(Player player, float p1);
 ```
+```
```

Comment:
Do you think you could remove this entire code block, add a proper description and provide a working example in Lua?

This code [example](https://github.com/citizenfx/natives/blob/master/HUD/EndTextCommandThefeedPostAward.md?plain=1#L36) shows how it should be written by the way.

You can also check out the contribution guide [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)